str/helper/vcf_combiner.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error, too-many-locals
```

Comment:
Do you know what is causing these errors? And do they definitely still occur? It doesn't seem like you have any import errors. The local variables count is quite low by default, but it's been overridden as 25 [here](https://github.com/populationgenomics/sv-workflows/blob/main/.pylintrc#L43), and you have way fewer than that

Reply:
sorry, this was copy and pasted from an old script, deleting now